### ISSUE TITLE
invokable -> invocable false positive?

### ISSUE THREAD

**Issue by justanotheranonymoususer** [crate-ci/typos#1065]
English is not my native language, but I saw `invokable` in code, and looks like both invokable and invocable are OK?
https://en.wiktionary.org/wiki/invocable
https://en.cppreference.com/w/cpp/utility/functional/invoke

**Comment by epage** [comment]
I wish wikitionary had citations.  Google always tries to correct `invokable` to `invocable`.  There is no varcon entry for this. Oxford English Dictionary does not recognize invokable.

Some dictionaries, like Firefox and Libreoffice, don't recognize either word.

**Comment by not-my-profile** [comment]
I think Google Ngram's is a good tool to judge whether something is a typo or just less common.

[invocable,invokable](https://books.google.com/ngrams/graph?content=invocable%2Cinvokable&year_start=2000&year_end=2019&corpus=en-2019&smoothing=3): it appears to be less common but common enough to be a spelling variant.

**Comment by justanotheranonymoususer** [comment]
Yeah, Googling for it shows that it's used in docs and identifiers of high profile companies such as Microsoft, Google and Oracle.

- https://learn.microsoft.com/en-us/previous-versions/dn910268(v=vs.85)
- https://guava.dev/releases/19.0/api/docs/com/google/common/reflect/Invokable.html
- https://docs.oracle.com/cd/E16162_01/apirefs.1112/e17493/oracle/ide/palette/Invokable.html
- https://docwiki.embarcadero.com/RADStudio/Athens/en/Understanding_Invokable_Interfaces
- http://jmock.org/javadoc/jmock-2.6.0/doc/org/jmock/api/Invokable.html
- https://laravel.com/docs/11.x/controllers

I think that calling it a typo is a stretch given that broad usage.

**Comment by justanotheranonymoususer** [comment]
I encountered a similar detection for cachable -> cacheable. cacheable seems more common, but cachable is widely used too, and wikitionary mentions it as alternative spelling:
https://en.wiktionary.org/wiki/cachable

**Comment by epage** [comment]
> Should I create a separate issue for this?

Please
- It muddies the waters to have multiple concurrent conversations
- This muddies the waters for the definition of done
